src/app/(proper_react)/(redesign)/(authenticated)/user/welcome/Onboarding.test.tsx

```diff
@@ -176,6 +176,74 @@ it("confirm dialog is showing on step 2", async () => {
   expect(screen.queryByText("Is this correct?")).not.toBeInTheDocument();
 });
 
+it("confirm dialog is showing mandatory user information on step 2", async () => {
```

Comment:
Btw, not a huge deal now since I believe the existing tests do this too, but generally, the idea is that test descriptions form a sentence that start with "it", where "it" refers to the general module under test or whatever's mentioned in the `describe` function, and "it confirm dialog is showing mandatory user information on step 2" is a bit of a weird sentence :sweat_smile: 

I believe the thought is to encourage people to describe behaviour in the test description (which already happens right now, so again, not a biggie). The symptoms that was in response to is test being labelled like "test confirm dialog mandatory fields", or even just "test confirm dialog", which tell the reader nothing at all.

Just a little context that might be helpful in the future :)

Reply:
I must admit this is more enjoyable to read. I’ve updated a couple of names in [f8fa487](https://github.com/mozilla/blurts-server/pull/4465/commits/f8fa48712f881ba71d493299f56ec40b57db2e9a).